client/my-sites/post-type-list/post-action-counts/index.jsx

```diff
@@ -1,5 +1,3 @@
-/* @format */
-
```

Comment:
This comment is not valid - it should have an extra asterisk like `/** @format */`.

This seems to break our automatic formatting setup pretty spectacularly.  There may be a problem with our pre-commit hook that doesn't run the formatting, and attempting to format this file generates a large diff that obscures the actual changes here and also mangles this comment in an interesting way:

```js
/**
 * /* @format
 *
 * @format
*/
```

So I think the best thing to do for now is remove the format tag.  We can re-add it and auto-format this file in a separate PR or just before merging this PR.

Reply:
I agree -- sorry 'bout that.